fastlane/Fastfile

```diff
@@ -110,6 +113,33 @@ lane :publish_release do |options|
 
   version = options[:version]
 
+  # Update Frameworks (WIP)
+  # update_frameworks
+
+  # Create Github Release
+  github_release = set_github_release(
+                     repository_name: "GetStream/stream-chat-swift",
+                     api_token: ENV["GITHUB_TOKEN"],
+                     name: version,
+                     tag_name: version,
+                     description: changes
+                     # upload_assets: ["Products/StreamChat.zip", "Products/StreamChatUI.zip", "Products/StreamChat-All.zip"]
+                   )
+  # The & operator makes sure truthy values are converted to bool
+  # and falsy (false and nil) values are converted to bool false
+  push_pods(sync: options[:sync] & true)
+
+  # Set tag
+  sh("git tag #{version}")
+
+  # Push tag to remote
+  push_to_git_remote(tags: true)
+
+  UI.success("Github release was created, please visit #{github_release["url"]} to see it")
+end
+
+desc "Update XCFrameworks and submit to the SPM repository"
```

Comment:
This is extracted logic to deal with the frameworks as part of the release. Github Actions struggles to deal with multiple repo's in the same action, I am likely to make this a new workflow.